pandemania/src/api/Toasts.js

```diff
@@ -0,0 +1,13 @@
+import { toast } from 'react-toastify';
+
+export const apiErrorToast = () => {
+	toast.error('Covid API id down, please try again later', { toastId: 'api-error' });
```

Comment:
"Is" instead "id"
And maybe covid 19 instead of covid.

Reply:
fixed